Duplicati/Library/Main/Controller.cs

```diff
@@ -865,6 +865,22 @@ private void ValidateOptions(ILogWriter log)
             if (!string.IsNullOrWhiteSpace(m_options.Prefix) && m_options.Prefix.Contains("-"))
                 throw new Interface.UserInformationException("The prefix cannot contain hyphens (-)");
 
+            //Check validity of retention-policy option value

```

Comment:
I think we should also check that the user does not have multiple options set here.
The `--keep-versions` option should not be allowed with `--retention-policy` as it is a bit unclear to the user what happens.